.circleci/config.yml

```diff
@@ -0,0 +1,119 @@
+# These environment variables must be set in CircleCI UI
+#
+# NPM_TOKEN - A valid NPM token for releases
+# GH_USER   - A GitHub username
+# GH_EMAIL  - The email linked to the username above
+# GH_TOKEN  - A GitHub token linked to the username above
```

Comment:
@muffinresearch it looks we'd need these `GH_*` env variables too. It's related to the `scripts/publish-rules` script.

Reply:
Thanks, those are added.

Reply:
Scope is public_repo